### PR TITLE
fix(types): explicit export path in package.json

### PR DESCRIPTION
**Pull request by BenjaminPalko**
### What did you fix? 
image
Fixed types not being exported properly through package.json for ESM projects

---

### Reproduction steps
Install warframe-items as a dependency on an ESM Typescript project (Webstorm)

---

### Considerations
- Does this contain a new dependency? **[No]**
- Does this introduce opinionated data formatting or manual data entry? **[No]**
- Does this pr include updated data files in a separate commit that can be reverted for a clean code-only pr? **[No]**
- Have I run the linter? **[Yes]**
- Is is a bug fix, feature request, or enhancement? **[Bug Fix]**

### CODE REVIEW

**Comment by TobiTenno** [comment]
Your description is misleading. This only applies to ESM projects.

File: package.json [hunk 1]
```diff
@@ -5,8 +5,9 @@
   "main": "index.js",
   "types": "index.d.ts",
   "exports": {
+    "types": "./index.d.ts",
     "import": "./index.mjs",
-    "require": "./index.js"
```
Comment (TobiTenno) [inline package.json:9]: Why? No thanks. Please don't remove an existing.

**Review by TobiTenno — CHANGES_REQUESTED** [review]

**Comment by BenjaminPalko** [comment]
> Your description is misleading. This only applies to ESM projects.

Ty, didn't realize it was only ESM.

**Review by TobiTenno — APPROVED** [review]

**Comment by TobiTenno** [comment]
uhhhh, looks like some weirdness with types... doubt it's specifically from you, but if you wanna take a look at that error and put a ticket in with the root cause and fix that, can't merge this until that's fixed (sorry)

**Pull request merged**